web/webpack.plugin.js

```diff
@@ -0,0 +1,207 @@
+const path = require('path');
+const DynamicRemotePlugin = require('@openshift/dynamic-plugin-sdk-webpack').DynamicRemotePlugin;
+const CSSMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const webpack = require('webpack');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+
+const BG_IMAGES_DIRNAME = 'assets';
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const pathTo = (relativePath) => path.resolve(__dirname, relativePath);
+
+/**
+ * Shared modules consumed and/or provided by this plugin.
+ *
+ * A host application typically provides some modules to its plugins. If an application
+ * provided module is configured as an eager singleton, we suggest using `import: false`
+ * to avoid bundling a fallback version of the module when building your plugin.
+ *
+ * Plugins may provide additional shared modules that can be consumed by other plugins.
+ *
+ * @see https://webpack.js.org/plugins/module-federation-plugin/#sharing-hints
+ */
+const pluginSharedModules = {
```

Comment:
For console dot, you also want to share the  `react-router-dom` dependency.

Reply:
Added